runtime/src/main/java/org/corfudb/runtime/BootstrapUtil.java

```diff
@@ -54,18 +55,22 @@ public static void bootstrap(@NonNull Layout layout,
      * @param layout Layout to bootstrap with
      */
     private static void bootstrapLayoutServer(IClientRouter router, Layout layout)
-            throws ExecutionException, InterruptedException, AlreadyBootstrappedException {
+            throws ExecutionException, InterruptedException, AlreadyBootstrappedException, TimeoutException {
         LayoutClient layoutClient = new LayoutClient(router, layout.getEpoch());
 
         try {
             CFUtils.getUninterruptibly(layoutClient.bootstrapLayout(layout),
-                    AlreadyBootstrappedException.class);
+                    AlreadyBootstrappedException.class, TimeoutException.class);
         } catch (AlreadyBootstrappedException abe) {
             if (!layoutClient.getLayout().get().equals(layout)) {
                 log.error("BootstrapUtil: Layout Server {}:{} already bootstrapped with different "
                         + "layout.", router.getHost(), router.getPort());
                 throw abe;
             }
+        } catch (TimeoutException toe){
+            log.error("BootstrapUtil: Layout Server {}:{} timeout occurred.",
+                    router.getHost(), router.getPort());
+            throw toe;
```

Comment:
I don't understand why we need to explicitly handle TimeoutException here. What about NetworkExceptions and other exceptions? Besides, the caller of this, namely BootstrapUtil.bootstrap() already logs and handles all exceptions: https://github.com/CorfuDB/CorfuDB/blob/master/runtime/src/main/java/org/corfudb/runtime/BootstrapUtil.java#L128-L134